src/nvim/screen.c

```diff
@@ -1731,6 +1740,53 @@ static int compute_foldcolumn(win_T *wp, int col)
   return fdc;
 }
 
+static int line_putchar(LineState *s, schar_T *dest, int maxcells, bool rl)
```

Comment:
common prefix (`screen_line_putchar`) helps a lot for understanding stacktraces, even for static functions.

Reply:
But this function does not operate on the screen, like the other `screen_` functions does, it operates on a line buffer. The prefix actually means something.

Reply:
well, ok. But the general idea is that following a stacktrace is easier when one sees the module prefix. Sometimes the prefix is meaningless but at least one knows the module. Not a big deal though.

Reply:
True, but currently it is a semantic prefix and not a module prefix. We could of course change this, but that would be reasonably done after (or maybe in) #8455.